R/assign.R

```diff
@@ -61,8 +61,11 @@ sdtm_assign <- function(tgt_dat = NULL,
   # Recode the raw variable following terminology.
   tgt_val <- ct_map(join_dat[[raw_var]], ct_spec = ct_spec, ct_clst = ct_clst)
 
+  # Current target values
+  cur_tgt_val <- join_dat[[tgt_var]] %||% as.vector(NA, mode = typeof(tgt_val))
+
   join_dat |>
-    mutate("{tgt_var}" := tgt_val) |> # nolint object_name_linter()
+    mutate("{tgt_var}" := dplyr::coalesce(cur_tgt_val, tgt_val)) |> # nolint object_name_linter()
```

Comment:
```suggestion
    dplyr::mutate({{tgt_var}} := dplyr::coalesce(cur_tgt_val, tgt_val)) |> # nolint object_name_linter()
```
@ramiromagno I tested the branch using the mock data I prepared for the PharmaSUG workshop, and it seems that the coalesced values are not correctly assigned to the target variable name tgt_var. I went in and debugged this function and realized it would work with {{}} symbol. Please consider the code suggestion above. Thank you!